Makefile

```diff
@@ -0,0 +1,43 @@
+.PHONY: all build test run bash tag push pull help
+
+NAME=echoapi
+NAMESPACE=quay.io/3scale
+VERSION=centos7-to-ocp
+LOCAL_IMAGE := $(NAME):$(VERSION)
+REMOTE_IMAGE := $(NAMESPACE)/$(LOCAL_IMAGE)
+
+MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+THISDIR_PATH := $(patsubst %/,%,$(abspath $(dir $(MKFILE_PATH))))
+PROJECT_PATH := $(patsubst %/,%,$(dir $(MKFILE_PATH)))
+
+
+all: build
+
+build: ## Build docker image with name LOCAL_IMAGE (NAME:VERSION).
+	docker build -f $(THISDIR_PATH)/Dockerfile -t $(LOCAL_IMAGE) $(PROJECT_PATH)
+
+test: ## Test built LOCAL_IMAGE (NAME:VERSION).
+	docker run --name $(VERSION) -t -p 9292:9292 -d $(LOCAL_IMAGE) 
```

Comment:
missing `-u 10000001` we want to test if it works fine in openshift (remember openshift uses a random UID)